packages/graphback-datasync/src/providers/createDataSyncMongoDbProvider.ts

```diff
@@ -1,20 +1,31 @@
 import { Db } from 'mongodb';
-import { ModelDefinition } from '@graphback/core';
+import { ModelDefinition, defaultTableNameTransform } from '@graphback/core';
 import { MongoDBDataProvider, GraphbackDataProvider } from "@graphback/runtime-mongo"
 import { isDataSyncModel } from '../util';
+import { PruneScheduler, PruneConfiguration, TTLMap } from "../scheduler";
 import { DataSyncMongoDBDataProvider } from './DatasyncMongoDBDataProvider';
 
 /**
  * Creates a new Data synchronisation data provider for MongoDb
  *
  * @param {Db} db - MongoDb connection
  */
-export function createDataSyncMongoDbProvider(db: Db): (...args: any[]) => GraphbackDataProvider {
+export function createDataSyncMongoDbProvider(db: Db, ttlMap?:TTLMap, pruneConfig: PruneConfiguration = {}): (...args: any[]) => GraphbackDataProvider {
+  const scheduler = new PruneScheduler(pruneConfig, db);
+  
   return (model: ModelDefinition): GraphbackDataProvider => {
     if (isDataSyncModel(model)) {
+
+      const modelName = model.graphqlType.name;
+      const modelTTL = ttlMap[modelName];
+      if (modelTTL >= 0) {
+        const collectionName = defaultTableNameTransform(modelName, 'to-db');
```

Comment:
```suggestion
        const collectionName = getTableName(model.graphqlType);
```

https://github.com/aerogear/graphback/blob/9df2aa574299db464b7f77293bb493f1f799c522/packages/graphback-core/src/db/buildModelTableMap.ts#L27

Reply:
Yup! Thanks!